src/Process/Runner/MutationTestingRunner.php

```diff
@@ -74,26 +74,34 @@ public function __construct(
     /**
      * @param Mutation[] $mutations
      */
-    public function run(array $mutations, int $threadCount, string $testFrameworkExtraOptions): void
+    public function run(iterable $mutations, int $threadCount, string $testFrameworkExtraOptions): void
     {
-        $this->eventDispatcher->dispatch(new MutantsCreationWasStarted(count($mutations)));
+        $this->eventDispatcher->dispatch(new MutantsCreationWasStarted(0));
+        $this->eventDispatcher->dispatch(new MutationTestingWasStarted(0));
```

Comment:
I kept these event in place in sake of backward compatibility with output formatters